.github/workflows/nightly.yml

```diff
@@ -6,9 +6,5 @@ on:
   workflow_dispatch:
 
 jobs:
-  test-e2e:
-    uses: ./.github/workflows/test-e2e.yml
-    secrets: inherit
-
   test:
     uses: ./.github/workflows/test.yml
```

Comment:
[proposal] can we delete nightly entirely because we already run test on every PR.

Reply:
yeah i'd be happy to delete nightly